eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentEvents.scala

```diff
@@ -251,11 +251,13 @@ object PaymentFailure {
 
 }
 
-case class PathFindingExperimentMetrics(amount: MilliSatoshi,
+case class PathFindingExperimentMetrics(paymentHash: ByteVector32,
+                                        amount: MilliSatoshi,
                                         fees: MilliSatoshi,
                                         status: String,
                                         duration: FiniteDuration,
                                         timestamp: TimestampMilli,
                                         isMultiPart: Boolean,
                                         experimentName: String,
-                                        recipientNodeId: PublicKey)
+                                        recipientNodeId: PublicKey,
+                                        routingHints: Seq[Seq[ExtraHop]])
```

Comment:
This would remove some boilerplate code and make us store `null` in database instead of an empty json list. It would also make the post-migration data consistent with migrated data.

```suggestion
                                        routingHints_opt: Option[Seq[Seq[ExtraHop]]])
```

Reply:
Done